.eslintrc.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
   plugins: ['@typescript-eslint', 'prettier', 'qunit'],
   settings: {
     node: {
+      allowModules: ['@glimmer/debug', '@glimmer/local-debug-flags'],
```

Comment:
FWIW, these _are_ actual modules that can be resolved. Why do we have to explicitly whitelist them?

Reply:
Is it because those packages aren’t published (and therefore not listed in the respective package.json dependencies)?

Reply:
it's imported in a few lib folders, so it would have to be a non-dev dependency, but since it's a private package that won't work. best solution I could come up with was this one.

Reply:
Ya, makes sense. We avoided this with `@glimmer/env` by actually publishing...